FileMapper/path_util.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Path Utilities
+ */
+var PathUtil =
+    {
```

Comment:
This goes in previous line, can you just run clang-format on the all the files?

Reply:
clang-format made this change earlier.
I tried running it again too and it doesn't change anything in this file.

Reply:
Fine then, we can leave it as is

Reply:
okay